ArduCopter/GCS_Mavlink.cpp

```diff
@@ -1244,14 +1244,11 @@ void GCS_MAVLINK_Copter::handleMessage(mavlink_message_t* msg)
                 case MAV_FRAME_GLOBAL:
                 case MAV_FRAME_GLOBAL_INT:
                 default:
-                    // pv_location_to_vector does not support absolute altitudes.
-                    // Convert the absolute altitude to a home-relative altitude before calling pv_location_to_vector
-                    loc.alt -= copter.ahrs.get_home().alt;
-                    loc.relative_alt = true;
```

Comment:
you need to keep this one otherwise alt will be in ALT_FRAME_ABSOLUTE and not ALT_FRAME_ABOVE_HOME

Reply:
Why is that a problem?  the call on `Location loc` will cope.